docs/hinsage.txt

```diff
@@ -37,20 +39,20 @@ Where:
 
 -  :math:`N(v)` is the neighbourhood of node :math:`v`
 
--  :math:`D_{\text{p\ }}\lbrack \cdot \rbrack`\ is a random dropout with probability :math:`p` applied to its argument vector.
+-  :math:`D_{p}\lbrack \cdot \rbrack`\ is a random dropout with probability :math:`p` applied to its argument vector.
 
 The number of trainable parameters in layer :math:`k` for the mean
 aggregator is
 
-   :math:`{{2d}_{k}}/2 \times d_{k - 1} + d_{k} = d_{k}d_{k - 1} + d_{k}` if concat=True, or
+   :math:`d_{k - 1} + d_{k} = d_{k}d_{k - 1} + d_{k}` if concat=True, or
```

Comment:
I think you accidently deleted ``{{2d}_{k}}/2 \times ` instead of wrapping it in `\frac`

```suggestion
   :math:`\frac{{2d}_{k}}{2} \times d_{k - 1} + d_{k} = d_{k}d_{k - 1} + d_{k}` if concat=True, or
```

Reply:
Oops, good catch! I think what I actually meant to do was remove the whole left hand side?
```suggestion
   :math:`d_{k}d_{k - 1} + d_{k}` if concat=True, or
```
since it seemed a bit unnecessary to me to write `2d_k/2 * d_k-1 + d_k = d_k d_k-1 + d_k` - I guess it was supposed to be showing how we arrived at the result, but without some additional explanation for how we arrive at the left-hand side in the first place, it kind of looked strange to me.

What do you think?

Reply:
Yeah true, it doesn't make a whole lotta sense just by itself. Sounds good to me :)